### ISSUE TITLE
Applying settings in web interface causes duplicates of weight data

### ISSUE THREAD

**Issue by yesrod** [yesrod/hoplite#6]
Ended up with this mess during troubleshooting of #5 - every time I attempt to save data back to the shmem, the weight data is duplicated.

```
pi@hoplite:~ $ sudo cat /dev/shm/hoplite
{
  "data": {
    "co2": [
      60
    ],
    "temp": -687,
    "weight": [
      [
        17446,
        -9628
      ],
      [
        6189,
        1295
      ],
      [
        17447,
        -9604
      ],
      [
        6192,
        1283
      ],
      [
        17443,
        -9657
      ],
      [
        6188,
        1250
      ],
      [
        17443,
        -9602
      ],
      [
        6187,
        1296
      ],
      [
        17444,
        -9624
      ],
      [
        6185,
        1332
      ]
    ]
  },
  "config": {
    "hx": [
      {
        "channels": {
          "A": {
            "co2": false,
            "name": "Not Quite Cherry Gose",
            "weight": 17391,
            "tare": 4,
            "volume": 18.9,
            "refunit": 20.958,
            "offset": 298472,
            "size": "corny"
          }
        },
        "pd_sck": 6,
        "dout": 5
      },
      {
        "channels": {
          "A": {
            "co2": true,
            "name": "CO2",
            "weight": 6185,
            "tare": 4.82,
            "volume": 2.27,
            "refunit": 21.401,
            "offset": 128516,
            "size": "custom"
          }
        },
        "pd_sck": 13,
        "dout": 12
      }
    ],
    "weight_mode": "as_kg_net"
  }
}
```

**Comment by yesrod** [comment]
This is not related to the fix for #5; reverting that change doesn't resolve the issue.  This is happening somewhere else.

**Comment by yesrod** [comment]
Fixed by clearing hx_handles whenever setup_all_kegs() is called.
